src/Tribe/Customizer/Global_Elements.php

```diff
@@ -129,7 +129,10 @@ public function setup() {
 		$description         = $views_v2_is_enabled ? '' : esc_html__( 'Options selected here will override what was selected in the "General Theme" section.', 'the-events-calendar' );
 
 		$this->defaults = [
-			'link_color' => '#141827',
+			'link_color'               => '#141827',
+			'background_color'         => '#e9e4f0', // TODO: use customizer setting
+			'event_title_color'        => '#009051', // TODO: use customizer setting
```

Comment:
* :warning: Array double arrow not aligned correctly; expected 5 space(s) between "'event_title_color'" and double arrow, but found 8.
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 5b288140565bbe5fd8512a7b7a28471f</sub>